### PR TITLE
Real-Time Applications with Go and ReactJS (Part 1)

### PR DESCRIPTION
**Pull request by fdrechsler**
Work in Progress

### CODE REVIEW

**Comment by fdrechsler** [comment]
@schaeferthomas I added the Section about our "real-time" discussion at the Top.

**Comment by fdrechsler** [comment]
Ready to Merge, Please Review :)

File: tutorials/real-time-apps-with-go-and-reactjs_part1.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+//......
+func main() {
+    http.HandleFunc("/listen", listenHandler)
+    //......
+}
+```
+
+The Arrow Syntax "<-" belongs to one of the core concepts of concurrency in golang: `channel`s, it blocks the routine until it receives data from a `channel`.
+
+> A `channel` in go is a typed conduit that can receive data`channel <- data` and data can be read from `data <- channel` Writing to or Reading from a Channel BLOCKS the subroutine. [Example](https://gobyexample.com/channels)
```
Comment (n-se) [inline tutorials/real-time-apps-with-go-and-reactjs_part1.md]:
```suggestion
> A `channel` in go is a typed conduit that can receive data `channel <- data` and data can be read from `data <- channel` Writing to or Reading from a Channel BLOCKS the subroutine. [Example](https://gobyexample.com/channels)
```

File: tutorials/real-time-apps-with-go-and-reactjs_part1.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+```go
+func sayHandler(w http.ResponseWriter, r *http.Request) {
+ w.Write([]byte(r.FormValue("name")))
+
+}
+```
+
+Test: `curl` (or open it in any web browser)
+
+```sdafsdf
```
Comment (n-se) [inline tutorials/real-time-apps-with-go-and-reactjs_part1.md]: What is this?

File: tutorials/real-time-apps-with-go-and-reactjs_part1.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+        }
+    }()
+    w.Write([]byte("ok"))
+```
+
+> pay attention to the `}()`: We are creating an instantly invoking function. 
+
+## Conclusion
+
+We just built a Realtime Chat app in 45 Lines of Go.
```
Comment (n-se) [inline tutorials/real-time-apps-with-go-and-reactjs_part1.md]:
```suggestion
We just built a real-time Chat app in 45 Lines of Go.
```

Comment (n-se) [reply tutorials/real-time-apps-with-go-and-reactjs_part1.md]: Or if a proper noun fits better, `Real-Time`

Comment (fdrechsler) [reply tutorials/real-time-apps-with-go-and-reactjs_part1.md]: Correct proper noun would be Real-time I guess. But it isn’t.

**Pull request merged**